minetest/minetest
# Add server url button to main menu

Add compact, short information about your PR for easier understanding:

This PR adds a button to the main menu for players to open the URL of the selected server.

- Goal of the PR
Add a button to open the URL.
- How does the PR work?
It adds a button and uses the `core.open_url_dialog` to display the URL and open it.
- Does it resolve any reported issue?
#15237
- Does this relate to a goal in [the roadmap](https://github.com/minetest/minetest/blob/master/doc/direction.md)?
2.3

## How to test

Open the menu and select a server, button should show up.
Press it and see if it opens the URL.

## Preview
![image](https://github.com/user-attachments/assets/e483785c-781d-46af-8861-8a22f2140e6e)
![image](https://github.com/user-attachments/assets/7e8ec5a0-5a1f-4eb9-8d4e-964914bebbf2)
